integrations/smart-sheet.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Smartsheet",
+  "auth": {
+    "authorizationURL": "https://app.smartsheet.com/b/authorize",
+    "tokenURL": "https://api.smartsheet.com/2.0/token",
+    "authType": "OAUTH2",
+    "tokenParams": { "grant_type": "authorization_code" },
+    "authorizationParams": {
+      "state":"random123"
```

Comment:
We probably have something to do about that. But it's out-of-scope for that PR 😅